ui/src/views/home/wallet-info.tsx

```diff
@@ -0,0 +1,19 @@
+import { useAccount } from 'wagmi';
+import { EnsInfo } from './ens-info';
+
+// Make sure that this component is wrapped with ConnectKitProvider
+export const WalletInfo = () => {
+  const { address, isConnecting, isDisconnected } = useAccount({
+    onConnect({ address, connector, isReconnected }) {
+      console.log('Connected to wallet', address, connector, isReconnected);
+    },
+  });
+  if (isConnecting) return <div>Connecting...</div>;
+  if (isDisconnected) return <div>Disconnected</div>;
+  return (
+    <>
+      <div>Connected Wallet: {address}</div>
+      {/* {address && <EnsInfo address={address} />} */}
+    </>
+  );

```

Comment:
The same comment about the component, it might change in the feature? For example adding a loading spinner or so? We could identify this with a TODO comment.